internal/infoblox/paging_ibclient.go

```diff
@@ -0,0 +1,60 @@
+package infoblox
+
+import (
+	"fmt"
+	"reflect"
+
+	ibclient "github.com/infobloxopen/infoblox-go-client/v2"
+)
+
+func PagingGetObject[T any](
+	c ibclient.IBConnector,
+	obj ibclient.IBObject,
+	ref string,
+	queryParams map[string]string,
+	res *[]T,
+) (err error) {
+
+	pagingResponse := pagingResponseStruct[T]{
+		NextPageId: "",
+		Result:     make([]T, 0),
+	}
+
+	//copy query params and update them
+	queryParamsCopy := map[string]string{}
+	for k, v := range queryParams {
+		queryParamsCopy[k] = v
+	}
+
+	queryParamsCopy["_return_as_object"] = "1"
+	queryParamsCopy["_paging"] = "1"
+	queryParamsCopy["_max_results"] = "1000"
```

Comment:
I tested against IBX with _max_results = "50", "100", but no effect on paging. Is there a minimum page size that needs to be specified?